vitest.ts

```diff
@@ -0,0 +1,11 @@
+import { 
+    describe as vitestDescribe, 
+    test as vitestTest, 
+    expect as vitestExpect 
+} from 'vitest';
+
+// Redirect Jest functions to Vitest
+globalThis.describe = vitestDescribe;
+globalThis.test = vitestTest;
+globalThis.it = vitestTest; // Alias for `it` if used in tests
+globalThis.expect = vitestExpect;
```

Comment:
some `hacky` way to get around the small API incompatibility between jest and vitest